app/helpers/payment_mailer.rb

```diff
@@ -0,0 +1,11 @@
+class PaymentMailer < ActionMailer::Base
+  layout 'mailer'
+  default from: Setting.mail_from
+  def self.default_url_options
+    Mailer.default_url_options
+  end  
+  def notify_payment(user)
+    
+    mail(to: user.mail, subject: "payment has been made")
```

Comment:
Email subject should specify details like other redmine notification emails. Also Email body should contain useful clickable information so that I get full details about what payment has been made and against what invoice. As well as total due, paid amount, and remaining etc.